src/controllers/controller.line.js

```diff
@@ -315,4 +315,19 @@ module.exports = DatasetController.extend({
 		model.borderWidth = valueOrDefault(options.hoverBorderWidth, options.borderWidth);
 		model.radius = valueOrDefault(options.hoverRadius, options.radius);
 	},
+
+	setDatasetHoverStyle: function() {
```

Comment:
Should this be `@protected` like `setHoverStyle` is?

Reply:
done